scripts/k8s-versions-test.sh

```diff
@@ -0,0 +1,346 @@
+#!/usr/bin/env bash
+# test application running in multiple versions of kubernetes ]
+
+# exit on unset vars
+set -u
+
+function add_hosts {
+        ENDPOINTSLIST=(127.0.0.1    localhost ml-api-adapter.local central-ledger.local account-lookup-service.local 
+                   account-lookup-service-admin.local quoting-service.local central-settlement-service.local 
+                   transaction-request-service.local central-settlement.local bulk-api-adapter.local 
+                   moja-simulator.local sim-payerfsp.local sim-payeefsp.local sim-testfsp1.local sim-testfsp2.local 
+                   sim-testfsp3.local sim-testfsp4.local mojaloop-simulators.local finance-portal.local
+                   operator-settlement.local settlement-management.local toolkit.local testing-toolkit-specapi.local
+                   admin-api-svc.local transfer-api-svc.local ) 
+
+        export ENDPOINTS=`echo ${ENDPOINTSLIST[*]}`
+
+        perl -p -i.bak -e 's/127\.0\.0\.1.*localhost.*$/$ENV{ENDPOINTS} /' /etc/hosts
+
+}
+
+function add_helm_repos { 
+  ## add the helm repos required to install and run ML and the v14 PoC
+        printf "==> add the helm repos required to install and run ML and the v14 PoC\n" 
+        su - vagrant -c "helm repo add mojaloop http://mojaloop.io/helm/repo/ > /dev/null 2>&1 "
+        su - vagrant -c "helm repo add kiwigrid https://kiwigrid.github.io  > /dev/null 2>&1 "
```

Comment:
You should only need a single repo here: `http://docs.mojaloop.io/charts/`

See https://github.com/mojaloop/charts#published-helm-chart-repo.

This is assuming you will be deploying the "example-backends". If you are deploying each backed-dependency separately, then you will need them.